dxr/plugins/python/tests/test_modules_packages/test_modules_packages.py

```diff
@@ -0,0 +1,49 @@
+from dxr.testing import DxrInstanceTestCase
+
```

Comment:
It won't because right now we don't support partial matches on dotted names. If you searched for `module:eee` it'd work, but since `snook.eee` is only a partial match it doesn't work.

A future PR adding partial match support would be great and is on my todo list. :D